pkgs/development/libraries/libgsignon-glib/default.nix

```diff
@@ -0,0 +1,67 @@
+{ stdenv, fetchFromGitLab, pkgconfig, automake, autoconf, libtool
+, glib, vala, gobjectIntrospection, gsignond, gettext, check
+, dbus, python3Packages, glibcLocales, docbook_xml_dtd_43
```

Comment:
You should not accept both `pythonPackages` and `python`, it can lead to mismatches.